src/css/imports/captions.less

```diff
@@ -47,11 +47,11 @@
 .jw-text-track-display {
     // Override vtt.js font-size so text resizes as the player size changes
     font-size: inherit;
-    line-height: 1.5em;
+    line-height: 1.5;
```

Comment:
That was intentional. The browser handles line-height better with unitless numbers. ([MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#Prefer_unitless_numbers_for_line-height_values))